src/server/transport/socketio.ts

```diff
@@ -17,76 +17,52 @@ import type {
 } from '../../master/master';
 import { getFilterPlayerView } from '../../master/filter-player-view';
 import type { Game, Server } from '../../types';
+import type { GenericPubSub } from './pubsub/generic-pub-sub';
+import { PubSubChannelIdNamespace } from './pubsub/generic-pub-sub';
+import type { IntermediateTransportData } from '../../master/master';
+import { InMemoryPubSub } from './pubsub/in-memory-pub-sub';
+import type { PubSubChannelId } from './pubsub/generic-pub-sub';
```

Comment:
Combine type imports from the same file.

```suggestion
import { PubSubChannelIdNamespace } from './pubsub/generic-pub-sub';
import type { IntermediateTransportData } from '../../master/master';
import { InMemoryPubSub } from './pubsub/in-memory-pub-sub';
import type { GenericPubSub, PubSubChannelId } from './pubsub/generic-pub-sub';
```

Reply:
Done